packages/toolkit/src/codewhisperer/models/constants.ts

```diff
@@ -297,7 +293,7 @@ export const transformByQCompletedMessage = 'Transformation completed'
 export const transformByQPartiallyCompletedMessage = 'Transformation partially completed'
 
 export const noPomXmlFoundMessage =
-    'We could not find a valid configuration file. We currently support Maven build tool and require a POM.xml in the root directory to identify build configurations. Be sure to also build your project.'
+    'None of your open Java projects are supported by Q Code Transformation. We were unable to find a pom.xml in any of your Java projects. We only support Java projects built on Maven at the moment.'
```

Comment:
Should this be "Amazon Q", I thought that marketing decision was that "Q" should never be presented without "Amazon".
```suggestion
    'None of your open Java projects are supported by Amazon Q Code Transformation. We were unable to find a pom.xml in any of your Java projects. We only support Java projects built on Maven at the moment.'
```

same for the other cases in this PR and all other code/messages.

Reply:
That's correct. Fixed.